tests/waveforms/cbc/test_cbc_waveforms.py

```diff
@@ -414,18 +690,87 @@ def test_phenom_p(
         hp_lal_data = hp_lal.data.data[lal_mask]
         hc_lal_data = hc_lal.data.data[lal_mask]
 
-        assert np.allclose(
-            1e21 * hp_lal_data.real, 1e21 * hp_ml4gw.real.numpy(), atol=2e-2
-        )
-        assert np.allclose(
-            1e21 * hp_lal_data.imag, 1e21 * hp_ml4gw.imag.numpy(), atol=2e-2
-        )
-        assert np.allclose(
-            1e21 * hc_lal_data.real, 1e21 * hc_ml4gw.real.numpy(), atol=2e-2
-        )
-        assert np.allclose(
-            1e21 * hc_lal_data.imag, 1e21 * hc_ml4gw.imag.numpy(), atol=2e-2
-        )
+        if request.config.getoption("--benchmark") == 0:
+            assert np.allclose(
+                1e21 * hp_lal_data.real,
+                1e21 * hp_ml4gw.real.numpy(),
+                atol=2e-2,
+            )
+            assert np.allclose(
+                1e21 * hp_lal_data.imag,
+                1e21 * hp_ml4gw.imag.numpy(),
+                atol=2e-2,
+            )
+            assert np.allclose(
+                1e21 * hc_lal_data.real,
+                1e21 * hc_ml4gw.real.numpy(),
+                atol=2e-2,
+            )
+            assert np.allclose(
+                1e21 * hc_lal_data.imag,
+                1e21 * hc_ml4gw.imag.numpy(),
+                atol=2e-2,
+            )
+
+        # If benchmarking calculate errors to record
+        # test_2 refers to differences between lal and ml4gw
```

Comment:
The difference between test 1 and test 2 is that distances are greater than 400 Mpc for test 1 and less than 400 Mpc for test 2. I'd suggest renaming test 1 and test 2 as "far" and "close"